server/db/seeds.rb

```diff
@@ -200,6 +188,24 @@
     directionality: nil,
     definition: 'Substrates are drugs which are metabolized by the protein they are interacting with.',
     reference: '<a href="https://www.fda.gov/drugs/drug-interactions-labeling/drug-development-and-drug-interactions-table-substrates-inhibitors-and-inducers">FDA - Table of Substrates, Inhibitors and Inducers</a>'
+  },
+  {
+    type: 'vaccine',
+    directionality: 0,
+    definition: 'In vaccine interactions, the drugs stimulate or restore an immune response to their target.',
+    reference: '<a href="https://www.cancer.gov/about-cancer/treatment/types/immunotherapy#q3">NCI - Cancer Vaccines</a>'
```

Comment:
Would like to revisit this style of data presentation when we cover the Sources page (in I think #170) because it's pretty janky to dump raw html into the database